elpis/engines/common/objects/interface.py

```diff
@@ -234,28 +232,23 @@ def list_models(self):
         return models
 
     def list_models_verbose(self):
-        if self.engine is None:
-            raise RuntimeError("Engine needs to be set to list models")
         models = []
         for hash_dir in os.listdir(f'{self.models_path}'):
             if not hash_dir.startswith('.'):
                 with self.models_path.joinpath(hash_dir,
                                                self.engine.model._config_file).open() as fin:
-                    model = json.load(fin)
-                    print('data', model)
-                    #  need to get results here
-                    model_info = {
-                        'name': model['name'],
-                        'dataset_name': model['dataset_name'],
-                        'pron_dict_name': model['pron_dict_name'],
-                        'status': model['status']
+                    data = json.load(fin)
+                    model = {
+                        'name': data['name'],
+                        'dataset_name': data['dataset_name'],
+                        'pron_dict_name': data['pron_dict_name']
                     }
-                    models.append(model_info)
+                    models.append(model)
         return models
 
     def new_transcription(self, tname):
         if self.engine is None:
-            raise RuntimeError("Engine needs to be set prior to transcription")
+            raise RuntimeError("Engine need to be set prior to transcription")
```

Comment:
This is a very small thing, but it seems to have gone from grammatically correct to not?

Reply:
fixed